frame/im-online/src/lib.rs

```diff
@@ -166,7 +166,7 @@ impl<BlockNumber: PartialEq + AtLeast32Bit + Copy> HeartbeatStatus<BlockNumber>
 	/// `session_index` - index of current session.
 	/// `now` - block at which the offchain worker is running.
 	///
-	/// This function will return `true` iff:
+	/// This function will return `true` if:
```

Comment:
```suggestion
	/// This function will return `true` iff:
```

"iff" is correct. https://en.wikipedia.org/wiki/If_and_only_if

Reply:
TIL, thanks!